website/blog/2020-03-24-version-0.62.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Announcing React Native 0.62 with Flipper
+author: Rick Hanlon
+authorTitle: React Native Core at Facebook
+authorURL: https://twitter.com/rickhanlonii
+authorImageURL: https://avatars3.githubusercontent.com/u/2440089?s=460&v=4
+authorTwitter: rickhanlonii
+category: announcements
+---
+
+Today we’re releasing React Native version 0.62 which includes support for Flipper by default.
+
+This release comes in the midst of a global pandemic. We’re releasing this version today to respect the work of hundreds of contributors who made this release possible and to prevent the release from falling too far behind master. Please be mindful of the reduced capacity of contributors to help with issues and prepare to delay upgrading if necessary.
+
+## Flipper by default
+
+[Flipper](https://fbflipper.com/) is a developer tool for debugging mobile apps. It’s popular in the Android and iOS communities, and in this release we’ve enabled support by default for new and existing React Native apps.
+
+Flipper provides the following features out of the box:
+
+- Network Inspector
+- Logout Inspector
+- Crash Reporter
+- Device logs
+
+There is also a marketplace of custom Flipper plugins you can install:
+
+[TODO: Flipper Image]
+
+For more information, check out the Flipper documentation here.
```

Comment:
The correct URL will be: https://fbflipper.com/docs/features/react-native.html

It is not yet online, but hopefully later today. I'll update here if it is online.